### ISSUE TITLE
Volume renderings are clipped in VR

### ISSUE THREAD

**Issue by markasselin** [KitwareMedical/SlicerOpenVR#14]
Below is a screenshot generated using the CT-Cardio. The volume rendering is clipping, where the controller, and the giant green sphere (which is a segment) is not.

![volumerendering-clipping](https://user-images.githubusercontent.com/17535566/33782903-816975e8-dc28-11e7-9c99-e9f20991d783.PNG)

**Comment by jcfr** [comment]
@jbvimort Could you try to reproduce this ?

**Comment by jcfr** [comment]
After investigating with @martinken, it looks like the issue is not specific to OpenVR. It can be reproduced in Slicer by enabling volume rendering and setting view angle to `130`.

```
getNode("vtkMRMLCameraNode1").GetCamera().SetViewAngle(130)
```

**Comment by mattjolley** [comment]
We are noticing this as well.  Any update?

**Comment by jcfr** [comment]
For reference, this is now tracked here: https://issues.slicer.org/view.php?id=4511

**Comment by jcfr** [comment]
Thanks @sankhesh  for figuring things out :+1:

**Comment by lassoan** [comment]
Yes, virtual reality works so well since the fix has been implemented!

Unfortunately, there is a small but in some use cases very distracting regression related to clipping - see details here: https://issues.slicer.org/view.php?id=4548. @sankhesh it would be great if you could have a look at it. Maybe it's a trivial fix.

**Comment by sankhesh** [comment]
@lassoan Sounds good. We can take a look to see if it is a quick fix.

**Comment by lassoan** [comment]
Thank you! I've sent standalone example to Aashish that reproduces the problem using one of the VTK tests.
